tclint.toml

```diff
@@ -35,8 +35,14 @@
     "src/utl/test/"
 ]
 
+ignore = [
+    "spaces-in-braces",
+    "unbraced-expr",
+]
+
 [style]
 indent = 2
 line-length = 100
 allow-aligned-sets = true
 indent-namespace-eval = false
+spaces-in-braces = true
```

Comment:
Even though this check is being ignored for now, this setting reduces the overall number of violations and will make it easier to transition to include this check.